setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-version = "0.0.9"
+version = "0.1.0"
```

Comment:
We don't have to go to `0.1.0`, we could also have `0.0.10`. I guess we don't have any strict checks for when to bump the two other numbers.

Reply:
Oh ok, fine by me.

Reply:
Done